CHANGELOG.md

```diff
@@ -26,7 +26,8 @@ This section is for maintaining a changelog for all breaking changes for the cli
 - Delete shape property ([#884](https://github.com/opensearch-project/opensearch-java/pull/885))
 
 ### Fixed
-- Fix version and build ([#254](https://github.com/opensearch-project/opensearch-java/pull/254))
```

Comment:
Put this back, don't remove people's changes ;)

Reply:
I put it back, sorry

Reply:
No stress ;)